### PR TITLE
Fix tests and linting

### PR DESCRIPTION
**Pull request by wcastand**
I figured i should make a PR for this before starting to add `--demo`.
chalk/chalk#210

If you want both in the same Pr let me know and i will add to this one.

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -12,12 +12,12 @@ const cli = meow(`
...
 
 	Options
-	  --template, -t  Style template. The \`~\` character negates the style.
+		--template, -t  Style template. The \`~\` character negates the style.
 
 	Examples
 	  $ chalk red bold 'Unicorns & Rainbows'
 	  $ chalk -t '{red.bold Unicorns & Rainbows}'
-	  $ chalk -t '{red.bold Dungeons and Dragons {~bold.blue (with added fairies)}}'
+		$ chalk -t '{red.bold Dungeons and Dragons {~bold.blue (with added fairies)}}'
```
Comment (Qix-) [inline cli.js]: Why the tabs here? Please remove them.

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 node_modules
+yarn.lock
```
Comment (Qix-) [inline .gitignore:2]: @sindresorhus thoughts on ignoring lockfiles? If :+1: then we also need to ignore `package-lock.json`.

Should probably also include `yarn-error.log` and `npm-debug.log` regardless.

File: test.js [hunk 3]
```diff
@@ -22,15 +22,15 @@ test('number', macro, {args: ['red', 'bold', '123', '--no-stdin']},
...
 test('template negation', templateMacro, '{red red {~red normal}}',
-	chalk.reset.red('red ') + chalk.reset('normal'));
+	chalk.red('red ') + chalk('normal'));
 
 test('template escaping #1', templateMacro, '{red hey\\} still red} not red',
-	chalk.reset.red('hey} still red') + chalk.reset(' not red'));
+	chalk.red('hey} still red') + chalk(' not red'));
 test('template escaping #2', templateMacro, '{red hey\\\\} not red',
-	chalk.reset.red('hey\\') + chalk.reset(' not red'));
+	chalk.red('hey\\') + chalk(' not red'));
```
Comment (Qix-) [inline test.js]: The places where you removed `.reset()` and left just `chalk` just need to be bare strings. Calling `chalk(someStr)` returns `someStr` unmodified.

**Review by Qix- — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (wcastand) [reply .gitignore:2]: I added the rest, if we need to remove it let me know. I looked at chalk and yarn.lock was ignored so i figured it was the same here.

If i'm wrong i will remove it :)

**Pull request merged**